main.py

```diff
@@ -1,10 +1,73 @@
 import init_django_orm  # noqa: F401
+import json
 
+from django.db import transaction
 from db.models import Race, Skill, Player, Guild
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/224108301-f161604c-8407-46a7-8009-fecc8bdaff45.png)

Reply:
Don't actually think you should do this within 2 fields, but for general practice, agree.